packages/next/src/server/app-render/app-render.tsx

```diff
@@ -155,6 +155,8 @@ import {
 import { CacheSignal } from './cache-signal'
 import { getTracedMetadata } from '../lib/trace/utils'
 
+import './clean-async-snapshot.external'
```

Comment:
consider moving this to all the same places that import from `'src/server/node-environment'`

We do this in the
* export worker (next build)
* build utils (though I'm not sure why for this one)
* static paths worker (generate static parmas)
* next-server (next start / next dev)
* router-server (not sure about this one)

Reply:
I think it should really be part of StaticGenerationStore so we pop back out to it but we just have to first be more diligent about how it's set up. E.g. not in unstable_cache and always outside RequestStore.